### ISSUE TITLE
neres get-state error

### ISSUE THREAD

**Issue by bookshelfdave** [glogiotatidis/neres#5]
* neres version: 0.3.0
* Python version: 2.7.13
* Operating System: MacOS Sierra 10.12.5

### Description

Trying to run `neres get-state` fails with an error:

```
AttributeError: 'module' object has no attribute 'SafeDumper'
```

### What I Did

```
$ neres get-state > state.yaml
Traceback (most recent call last):
  File "/usr/local/bin/neres", line 11, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/neres/cli.py", line 427, in get_state
    Dumper=yamlordereddictloader.SafeDumper,
AttributeError: 'module' object has no attribute 'SafeDumper'
```

**Comment by glogiotatidis** [comment]
Thanks for reporting @metadave. I guess that while trying to fix #4  you installed yamlorderedidctloader from pypi but we need the GitHub version. 

This should be fixed in 1149cc7

**Comment by bookshelfdave** [comment]
I pulled the latest, the same error occurs in python 2.7 or python 3.6 (w/ virtualenv).

**Comment by glogiotatidis** [comment]
Ok that was an interesting one involving unreleased python packages, setup.py and requirements.txt.

I think I fixed it with aae9807

Released 0.3.2 with the fix. Can you please try again @metadave ?

**Comment by bookshelfdave** [comment]
it works, thanks @glogiotatidis
